tenseal/cpp/tensors/tensor_storage.h

```diff
@@ -377,6 +380,23 @@ class TensorStorage {
         xt::from_json(nlohmann::json::parse(buf), _data);
     }
 
+    TensorStorage<dtype_t> subscript(
+        const vector<pair<size_t, size_t>>& pairs) {
+        xt::xstrided_slice_vector slices;
+        std::vector<size_t> shape = this->shape();
+        int index = 0;
+        for (auto pair : pairs) {
```

Comment:
While it is not critical here, usually is better to use `auto&` to prevent unnecessary copies in the loop.